src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+import javafx.application.Application;
+
+/**
+ * A launcher class to workaround classpath issues.
+ */
+public class Launcher {
+    public static void main(String[] args) {
+        Application.launch(Main.class, args);
+    }
+}
```

Comment:
Maybe providing further clarification on the specific classpath issues you mentioned in your JavaDoc would help to enhance understanding for future developers or reader reviewing your code.

Reply:
I have provided the clarification on the specific class path issues so other developers can review my code with improved readability.